### DISCUSSION TITLE
Using Reshade PostProcessors in LossLess

### DISCUSSION THREAD

**Discussion by defencedog** [mifi/lossless-cut#1201] [Ideas]
I am avid user of [Reshade ](https://reshade.me/) in gaming be it on Linux or Windows. Can we use its marvelous graphic post-processing ability in LossLess?

I thought LossLess will be using OpenGL or DirectX ...& tried installing Reshade but it didnt help! In DirectX case it broke the application

**Comment by mifi** [comment]
unfortunately not, because shading is not a lossless operaiton
